docs/third-party-developers/extensibility/data-store/checkout.md

```diff
@@ -0,0 +1,116 @@
+# wc/store/checkout
+
+## Table of Contents
+
+-   [Selectors](#selectors)
+    -   [getCustomerId](#getcustomerid)
+    -   [getCheckoutState](#getcheckoutstate)
+    -   [getOrderNotes](#getordernotes)
+    -   [hasError](#haserror)
+    -   [hasOrder](#hasorder)
+    -   [isAfterProcessing](#isafterprocessing)
+    -   [isBeforeProcessing](#isbeforeprocessing)
+    -   [isCalculating](#iscalculating)
+    -   [isComplete](#iscomplete)
+    -   [isIdle](#isidle)
+    -   [isProcessing](#isprocessing)
+
+## Selectors
+
+### getCustomerId
+
+Returns the ID of the customer.
+
+#### _Returns_
+
+-   `number`: ID of the customer.
+
+### getCheckoutState
+
+Returns the checkout state.
+
+#### _Returns_
+
+-   _state_ `Object`: Checkout state.
+
+### getOrderNotes
+
+Returns the order notes.
+
+#### _Returns_
+
+-   `string`: Order notes.
+
+### hasError
+
+Returns true if an error occurred, and false otherwise.
+
+#### _Returns_
+
+-   `boolean`: Whether an error occurred.
+
+### hasOrder
+
+Returns true if an order exists, and false otherwise.
+
+#### _Returns_
+
+-   `boolean`: Whether an order occurred.
+
+### isAfterProcessing
+
+Returns true if an order had just been processed, and false otherwise.
+
+#### _Returns_
+
+-   `boolean`: Whether an order had just been processed.
+
+### isBeforeProcessing
+
+Returns true if an order is about to be processed, and false otherwise.
+
+#### _Returns_
+
+-   `boolean`: Whether an order is about to be processed.
+
+### isCalculating
+
+Returns true if any of the totals, taxes, shipping, etc. need to be calculated, and false otherwise.
```

Comment:
Is this correct? I think this is true when there's an in-flight request to update these values, isn't it?

Reply:
I'm not 100% sure about that. That function is defined here: https://github.com/woocommerce/woocommerce-blocks/blob/361b48b6220f57fab8a62b9a9272757530d6f14a/assets/js/data/checkout/selectors.ts#L43-L45

And `calculatingCount` holds the following comment: https://github.com/woocommerce/woocommerce-blocks/blob/361b48b6220f57fab8a62b9a9272757530d6f14a/assets/js/data/checkout/default-state.ts#L15-L16

That's why I thought that this function tells whether some calculations in the checkout are required. That said, I might have worded it poorly. I'll change it according to your suggestion.